ui/src/app/collection/detail/manage-collection.component.spec.ts

```diff
@@ -302,16 +305,9 @@ describe("ManageCollectionComponent", () => {
       action.callback?.(action)
       expect(action.visible?.()).toBeTruthy()  // !== PublishStatus.Archived  && !== PublishStatus.Deleted
 
-      action = actions[4]
```

Comment:
Do you remember why this is being removed?

Reply:
was removed because now an if condition was prepended to fill the list of buttons displayed by the button (Archive Collection or Unarchive Collection)

before the list was 5 now it is 4

I found it easier to handle it that way.

![image](https://user-images.githubusercontent.com/103442937/188906600-e24ffb37-818a-46ca-8c05-9122fdfe251e.png)